frontend/src/components/ViewActivities/activity.js

```diff
@@ -1,19 +1,34 @@
 import React from 'react';
 import Card from 'react-bootstrap/Card';
-import * as utils from '../Utils/utils.js';
-import * as DBUTILS from '../../constants/dbconstants.js'
+import * as time from '../Utils/time.js';
+import * as DB from '../../constants/database.js'
 import '../../styles/activities.css';
+import Accordion from 'react-bootstrap/Accordion';
 
+/**
+ * A single activity. 
+ * 
+ * @param {Object} props This component expects the following props:
+ * - `activity` {Object} The activity to display.
```

Comment:
Nit: Redundant type for activity listed?

Reply:
I thought we were adding the prop type. whoopsie.